controllers/movies.js

```diff
@@ -0,0 +1,50 @@
+const movies = require('../movies')
+
+const getAllMovies = (request, response) => {
+  return response.send(movies)
+}
+
+const getMoviesByDirectors = ('/:director', (request, response) => {
```

Comment:
Controller functions get passed the request and response, not the route. The `app.get` already took care of the route. This causes the route to throw an error because your function is misdefined.

![image](https://user-images.githubusercontent.com/5775716/80131031-99ff6a00-8567-11ea-9e37-4c54ad459a08.png)